.github/workflows/ci.yml

```diff
@@ -12,8 +12,9 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        java: [ '8', '17' ]
+        java: [ '11', '17' ]
```

Comment:
So actually looking at the Bloop repo, this doesn't run on 8 at all for the gradle plugin and it's actually problemantic due to some of the tests. For now I'm going to switch this back to how it is in the Bloop repo, and then we can re-address later if need be.